templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>The MPH dir should be the same as the bin_file_dir, I think this will help to keep things organized moving forward, that was a change in my most recent cleanup PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>